lib/mayaUsd/ufe/UsdUndoMaterialCommands.cpp

```diff
@@ -96,7 +163,8 @@ bool _BindMaterialCompatiblePrim(const UsdPrim& usdPrim)
         // material or a shader.
         return false;
     }
-    if (UsdGeomScope(usdPrim) && usdPrim.GetName() == kDefaultMaterialScopeName) {
+    if (UsdGeomScope(usdPrim)
+        && usdPrim.GetName() == UsdMayaJobExportArgs::GetDefaultMaterialsScopeName()) {
```

Comment:
Use the resolved materials scope name instead of the hardcoded value. This was probably an oversight when moving the default materials scope name into UsdMayaJobExportArgs.